doc/apidoc/smac.epm.base_imputor.rst

```diff
@@ -0,0 +1,7 @@
+smac\.epm\.base\_imputor module
+===============================
+
+.. automodule:: smac.epm.base_imputor
```

Comment:
@KEggensperger could you please extend the docstring of the `BaseImputor` class?